inc/preimport.class.php

```diff
@@ -162,7 +162,7 @@ static function getSupplierPost($suppliername, $compSerial, $otherserial = null)
       if (!empty($suppliername)) {
          $supplierclass = "PluginManufacturersimports" . $suppliername;
          $supplier      = new $supplierclass();
-         $infos         = $supplier->getSupplierInfo($compSerial, $otherserial);
+         $infos         = $supplier->getSupplierInfo($compSerial, $otherserial, $supplierKey, $supplierUrl, $supplierSecret);
```

Comment:
The variables `$supplierKey, $supplierUrl, $supplierSecret` are not defined, so I think there is a problem